test/k8sT/manifests/bookinfo-v1-istio.yaml

```diff
@@ -128,17 +128,20 @@ spec:
         resources:
           limits:
             cpu: "2"
-            memory: 128Mi
+            memory: 1Gi
           requests:
             cpu: 100m
             memory: 128Mi
         securityContext:
           capabilities:
             add:
             - NET_ADMIN
+            - SYS_ADMIN
```

Comment:
@jrajahalme can you remove this so we can merge it?

Reply:
Isn't this the core requirement of this PR? Ie without it the bpf lookups can't be done from Istio sidecar?

Reply:
@joestringer AFAIK the conclusion of the weekly meeting was to drop this so we can merge the istio 1.1.7 support and return to this requirement later.

Reply:
Done.